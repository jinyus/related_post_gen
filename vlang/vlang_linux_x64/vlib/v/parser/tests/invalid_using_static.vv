fn main() {
	if static true {
		println(true)
	}
}
