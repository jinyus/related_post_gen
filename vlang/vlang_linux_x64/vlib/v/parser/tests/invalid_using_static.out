vlib/v/parser/tests/invalid_using_static.vv:2:5: error: the `static` keyword is invalid here
    1 | fn main() {
    2 |     if static true {
      |        ~~~~~~
    3 |         println(true)
    4 |     }
